src/Illuminate/Foundation/Bootstrap/DetectEnvironment.php

```diff
@@ -16,6 +16,11 @@ public function bootstrap(Application $app)
 	{
 		try
 		{
+			if( "mutable" == getenv('VAR_MUT') )
```

Comment:
Sorry, that's a habit I developed for combating accidental assignments like this:

```php
if( $foo = "bar" )
{
	// do stuff
}
```

I'll commit a fix that now.